gordo_components/server/server.py

```diff
@@ -123,10 +131,56 @@ def base_healthcheck():
     return app
 
 
-def run_server(host: str = "0.0.0.0", port: int = 5555, debug: bool = False):
-    app = build_app()
-    app.run(host, port, debug=debug, threaded=False)
+def run_server(
+    host: str, port: int, workers: int, worker_connections: int, log_level: str
+):
+    """
+    Run application with Gunicorn server using Gevent Async workers
+
+    Parameters
+    ----------
+    host: str
+        The host to run the server on.
+    port: int
+        The port to run the server on.
+    workers: int
+        The number of worker processes for handling requests.
+    worker_connections: int
+        The maximum number of simultaneous clients per worker process.
+    log_level: str
+        The log level for the `gunicorn` webserver. Valid log level names can be found
+        in the [gunicorn documentation](http://docs.gunicorn.org/en/stable/settings.html#loglevel).
+    """
 
+    cmd = [
+        "gunicorn",
+        "--bind",
+        f"{host}:{port}",
+        "--log-level",
+        log_level,
```

Comment:
Just a note here, you've set `case_sensitive=False` I assume that is true and you won't need to set `.lower()` or anything here?